compute_endpoint/globus_compute_endpoint/endpoint/endpoint_manager.py

```diff
@@ -1014,14 +1005,19 @@ def cmd_start_endpoint(
         udir, uid, gid = user_record.pw_dir, user_record.pw_uid, user_record.pw_gid
         uname = user_record.pw_name
 
-        if not self._allow_same_user:
+        if self.identity_mapper and not self._allow_same_user:
             p_uname = self._mu_user.pw_name
             if uname == p_uname or uid == os.getuid():
                 raise InvalidUserError(
-                    "Requested UID is same as Manager Endpoint UID, but configuration"
-                    " has not been marked to allow the Manager Endpoint UID to process"
-                    " tasks.  To allow the same UID to also run tasks, consider using"
-                    " a non-root user or removing privileges from the UID."
+                    "Requested UID is same as Manager Endpoint UID on a user-mapped"
+                    " Manager Endpoint. To allow the same UID to run tasks, consider: "
+                    "\n * using a non-root user,"
+                    "\n * removing privileges from the UID,"
+                    "\n * removing the identity mapping configuration file"
+                    f" ({self.identity_mapper.config_path}), or"
+                    "\n * setting `force_mu_allow_same_user: true` in the MEP config"
+                    f" ({Endpoint._config_file_path(self.conf_dir)})."
```

Comment:
I was going to say "This is the first time we'll have documented `force_mu_allow_same_user`."  Turns out I have a bad memory.  :disappointed:

Accordingly, before I checked myself, I was going to suggest updating this flag to something `mep` (e.g., `allow_mep_uep_same_user`).

Hrm.